lib/active_shipping/carriers/canada_post_pws.rb

```diff
@@ -294,6 +294,16 @@ def parse_rates_response(response, origin, destination)
       CPPWSRateResponse.new(true, "", {}, :rates => rates)
     end
 
+    def price_from_node(node, exclude_tax)
+      price = node.at('price-details/due').text
+      return price unless exclude_tax
+      taxes = 0
+      node.at('price-details/taxes').children.each do |node|
+        taxes += Package.cents_from(node.text) if node.elem?
```

Comment:
From some reason it thinks `taxes` has 7 children (counts newline in the xml response as a child). These newlines are of type `Text` where as the actual tax xml fields (gst,pst,hst) are of type `Element`, using `node.elem?` I skip over the newlines.